test_files/sickle/enum.ts

```diff
@@ -0,0 +1,3 @@
+enum FooEnum {XYZ, PI = 3.14159}
+/** @export */ FooEnum.XYZ;
```

Comment:
Could we generate this? I think that should (a) tell Closure what's going on and (b) make sure the value is available after minification and (c) allow the constant (`XYZ`) to be renamed.

```
(<any>FooEnum).XYZ = 1;
(<any>FooEnum).PI = 3.14159;
/** @type {FooEnum} */
FooEnum.X;
/** @type {FooEnum} */
FooEnum.PI;
```